.agents/skills/fern-recipe-page/scripts/check_recipe_page.py

```
@@ -0,0 +1,474 @@
+
+
+def catalog_targets(slug: str):
+    """(gpu, framework, topology, workload) per shipped target, from the catalog."""
+    for path in sorted((CATALOG / "recipes").glob("*.yaml")):
+        text = read(path)
+        page = re.search(r"^page:\s*(\S+)", text, re.M)
+        if not page or Path(page.group(1)).stem != slug:
+            continue
+        rows = []
+        for block in re.split(r"\n- id: ", text)[1:]:
+            grab = lambda pat: (re.search(pat, block).group(1) if re.search(pat, block) else None)
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `grab` as a `def`

(E731)